js/okcoinusd.js

```diff
@@ -623,17 +623,29 @@ module.exports = class okcoinusd extends Exchange {
         return { 'url': url, 'method': method, 'body': body, 'headers': headers };
     }
 
-    async request (path, api = 'public', method = 'GET', params = {}, headers = undefined, body = undefined) {
-        let response = await this.fetch2 (path, api, method, params, headers, body);
+    handleErrors (code, reason, url, method, headers, body) {
+        if (this.verbose)
+            console.log (this.id, method, url, code, reason, body ? ("\nResponse:\n" + body) : '');
```

Comment:
This is the last line it complains about, namely the conditional. Apart from it, looks clean. I'll fix it, you don't have to bother.

Reply:
Too late )

Actually, I was going to move it to the base class anyway. We can do it once there before calling `haldleErrors` instead of doing it in every impl. Ok?

Reply:
I support the proposal, less duplication == better.